builder/service.go

```diff
@@ -103,13 +104,17 @@ func getRelayConfig(endpoint string) (RelayConfig, error) {
 			if err != nil {
 				log.Info("invalid compliance list config for relay", "endpoint", endpoint, "err", err)
 			}
+			if complianceListsEnabled {
+				authHeader = bCfg.BloxrouteAuthHeader
+			}
```

Comment:
I think you can pass the auth header string into the function instead of `bCfg` and set directly on line 117